classes/submission/reviewAssignment/ReviewAssignmentDAO.inc.php

```diff
@@ -376,15 +375,21 @@ function insertObject($reviewAssignment) {
 		);
 
 		$reviewAssignment->setId($this->getInsertId());
-		return $reviewAssignment->getId();
+
+		// Update review stage status whenever a review assignment is changed
+		if ($result) {
```

Comment:
Ah, I see the `$dieOnError` and `fatal()` call in `DAO::update()` now. Removed the unnecessary result check: https://github.com/NateWr/pkp-lib/commit/74272c974be8fdfda030d785facfd8a60d6bd6fc